### DISCUSSION TITLE
Information on PSMs, MS/MS spectra information

### DISCUSSION THREAD

**Discussion by vanyabangera** [vdemichev/DiaNN#852] [Q&A]
Hello,

I searched a data using DIA-NN, but I was unable to find the number of PSMs, MS/MS spectra information. Can I get that information? If yes, could you please guide me on how or where to find it?

Thank you 
Vanya K N

**Comment by vdemichev** [comment]
Hi Vanya,

The number of lines in the main DIA-NN report is the number of identified PSMs at the specified FDR threshold.

Best,
Vadim

**Comment by vanyabangera** [comment]
Thank You, Vadim.

Kind Regards,
Vanya K N
+91 9148523213
